cleanup/cleanup.sh

```diff
@@ -94,6 +94,7 @@ get_deletion_candidates() {
 }
 
 main() {
+    echo 'Deprecated: Please use appuio:image-cleanup.'
```

Comment:
`Please use appuio:image-cleanup.` This is very confusing for me. Is it referencing a docker image on Docker hub, or a github project? I would change it to `Please use github.com/appuio/image-cleanup.`